config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
-if ENV['RAILS_USE_MEMORY_STORE'] || (!Rails.env.development? && !Rails.env.production?)
+if ENV['RAILS_USE_MEMORY_STORE'] || TokenStore.use_memory?
```

Comment:
Ah, I overlooked the fact that this affects UI sessions. I will remove that commit from batch.

Thanks @Fryguy for keeping closer eye!